ASIM/dev/Delete-SentinelFunction/Delete-LASavedFunction.ps1

```diff
@@ -0,0 +1,107 @@
+    <#
+        .SYNOPSIS
+        Deletes saved searches from Log Analytics workspace
```

Comment:
Per the doc and function name I assume no list option yet. Correct?

Reply:
We need to figure out how do we call and document "functions" or saved searches. Not sure what's right.

Reply:
yes. I agree. sorry for inconsistency

Reply:
actually this version of the script only works with functions, since it is looking for the functionalias